backend/src/router/session.go

```diff
@@ -226,3 +226,17 @@ func updateSession(w http.ResponseWriter, r *http.Request) {
 		})
 	}
 }
+
+func deleteSession(w http.ResponseWriter, r *http.Request) {
+
+	params := mux.Vars(r)
+	sessionID, _ := primitive.ObjectIDFromHex(params["id"])
+
+	deletedSession, err := mongodb.Sessions.DeleteSession(sessionID)
+	if err != nil {
+		http.Error(w, "Error deleting session: " + err.Error(), http.StatusNotFound)
+		return
+	}
+
+	json.NewEnconder(w).Encode(deletedSession)
```

Comment:
```suggestion
	json.NewEncoder(w).Encode(deletedSession)
```
Probably a typo because the function `NewEnconder` does not exist.